onyo/lib/commands.py

```diff
@@ -1084,6 +1084,67 @@ def onyo_rm(inventory: Inventory,
     ui.print('Nothing was deleted.')
 
 
+@raise_on_inventory_state
+def onyo_rmdir(inventory: Inventory,
+               dirs: list[Path] | Path,
+               message: str | None = None,
+               auto_message: bool | None = None) -> None:
+    r"""Delete empty directories or convert empty Asset Directories into Asset Files.
+
+    If the directory does not exist, the path is protected, or the asset is
+    already an Asset File, then an error is raised nothing is modified.
+
+    Parameters
+    ----------
+    inventory
+        The Inventory in which to delete directories or convert asset directories.
+    dirs
+        Paths of directories to delete or convert.
+    message
+        Commit message to append to the auto-generated message.
+    auto_message
+        Generate a commit-message subject line.
+        If ``None``, lookup the config value from ``onyo.commit.auto-message``.
+
+    Raises
+    ------
+    ValueError
+        ``dirs`` is empty.
+    """
+
+    if auto_message is None:
+        auto_message = inventory.repo.auto_message
+
+    if not dirs:
+        raise ValueError("At least one directory path must be specified.")
+
+    dirs = [dirs] if not isinstance(dirs, list) else dirs
+    for d in deduplicate(dirs):  # pyre-ignore[16]
+        try:
+            inventory.remove_directory(Item(d, repo=inventory.repo), recursive=False)
```

Comment:
One more thing ;-)

```suggestion
            inventory.remove_directory(inventory.get_item(d), recursive=False)
```

Reply:
Ahh yes. Thank you. Copy-paste from a different age of onyo. ;-)